textdb/textdb-angular-gui/app/sidebar/side-bar.component.ts

```diff
@@ -86,6 +94,25 @@ export class SideBarComponent {
         this.ModalOpen();
 
       });
+
+    currentDataService.metadataRetrieved$.subscribe(
+      data => {
+        //TODO:: show attributes according to the source table.
```

Comment:
Change this in the future: when table name changes, attributes should also change